<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPantsH</class>
 <widget class="QDialog" name="DialogPantsH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1060</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1060</width>
    <height>664</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>款式设计 - H型</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(232, 94, 35);
font: 9pt &quot;微软雅黑&quot;;
}
QPushButton:hover{
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.299, y1:0, x2:1, y2:0.00568182, stop:0.19403 rgba(255, 105, 46, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
font: 10pt &quot;微软雅黑&quot;;
}
*/</string>
  </property>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>H型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNext">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelString">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择款式：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>431</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="255">
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>664</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedPages">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>471</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_12"/>
    <widget class="QWidget" name="page_13"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="sang1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>前褶省</string>
   </property>
  </widget>
  <widget class="QPushButton" name="waistPostion">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>腰位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pocket2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>后袋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="waistHead1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>前腰头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pantsLength">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>裤长</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pantsLoop">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>裤袢</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pantsFoot">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>540</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>裤脚</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sang2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>后褶省</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pocket1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>前袋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="door">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>门襟</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;微软雅黑&quot;;
color: rgb(153, 153, 153);</string>
   </property>
   <property name="text">
    <string>W:向上选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;微软雅黑&quot;;
color: rgb(153, 153, 153);</string>
   </property>
   <property name="text">
    <string>S:向下选择</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>491</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(98, 98, 98);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="waistHead2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>后腰头</string>
   </property>
  </widget>
  <action name="pageDown">
   <property name="text">
    <string>下一部件</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="pageUp">
   <property name="text">
    <string>上一部件</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>labelText</zorder>
  <zorder>buttonNext</zorder>
  <zorder>labelString</zorder>
  <zorder>graphicsView</zorder>
  <zorder>sang1</zorder>
  <zorder>waistPostion</zorder>
  <zorder>pocket2</zorder>
  <zorder>waistHead1</zorder>
  <zorder>pantsLength</zorder>
  <zorder>pantsLoop</zorder>
  <zorder>pantsFoot</zorder>
  <zorder>sang2</zorder>
  <zorder>pocket1</zorder>
  <zorder>door</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>waistHead2</zorder>
 </widget>
 <tabstops>
  <tabstop>pantsLength</tabstop>
  <tabstop>waistPostion</tabstop>
  <tabstop>waistHead1</tabstop>
  <tabstop>pantsLoop</tabstop>
  <tabstop>door</tabstop>
  <tabstop>sang1</tabstop>
  <tabstop>pocket1</tabstop>
  <tabstop>sang2</tabstop>
  <tabstop>pocket2</tabstop>
  <tabstop>pantsFoot</tabstop>
  <tabstop>buttonNext</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
